webpack.config.babel.js

```diff
@@ -14,7 +14,10 @@ export default {
     loaders: [{
       test: /\.js$/, exclude: /node_modules/, loader: 'babel'
     }, {
-      test: /\.svg$/, loader: 'raw'
+      test: /\.svg$/, loader: 'raw', exclude: /node_modules/
+    }, {
+      test: /\.scss$/, exclude: /node_modules/,
+      loaders: ['style?insertAt=top', 'css', 'sass']
```

Comment:
we insert the stylesheet at the top of the webpage so that user styles will be on top of it